### PR TITLE
WIP: integrate fuzz testing

### PR DESCRIPTION
**Pull request by nullstyle**

### CODE REVIEW

File: test/fuzz/Rakefile [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+require 'pry'
+require 'faraday'
+require 'json'
+require 'fileutils'
+require 'base64'
+
+HORIZON = "https://horizon.stellar.org"
+HTTP = Faraday.new(url: HORIZON) 
+
+desc "tasks for the TransactionEnvelope fuzz test"
```
Comment (bartekn) [inline test/fuzz/Rakefile]: Why not golang?

Comment (nullstyle) [inline test/fuzz/Rakefile]: Ease of dev...  I just copy pasted from the XDR rakefile.  I've got a half written golang version that I'll finish before I remove the "WIP" label :)

**Comment by howardtw** [comment]
@bartekn this one looks pretty outdated, and it looks safe to close it. Echo?

**Comment by bartekn** [comment]
@howardtw sounds good! I don't remember the context here, but it seems that a better place for fuzz testing XDR would be in `go-xdr` package.

**Pull request closed without merging**